docs/daemon/cli/server.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 2
+sidebar_label: "Server"
+id: daemon-cmdline-server
+title: Server
+description: Kwild node Command Line
+slug: /daemon/cmdline/server
+---
+
+Kwild server commands provides a way to interact and operate kwild server.
```

Comment:
`kwild` server command are used to start and stop the Kwil daemon.

Also, I think we should create a command for stopping.  Right now, `kwild` starts and blocks the CLI.  However, this is a bit of a misnomer, since its a daemon.  We should have `kwild start` and `kwild stop`, both of which do not block (IMO, but we can discuss more as a team)

Reply:
I agree. Remember discussing about this with Gavin when I introduced start command. Right now, the problem i see is,  the way to idenitify the kwild process to shut it down. If its a systemd service, we can identify it using the service name, but clients can have their own aliases for kwild, so we probably shouldn't assume to grep for kwild  or kwild server start to find the pid.